test/system/README.md

```diff
@@ -1,43 +1,24 @@
 # System tests
 
 These tests are built with BATS (Bash Automated Testing System). They are
-strongly influenced by libpod project.
+strongly influenced by the [libpod](https://github.com/containers/libpod) project.
 
-The tests are meant to ensure that toolbox's functionality remains stable
+The tests are meant to ensure that Toolbox's functionality remains stable
 throughout updates of both Toolbox and Podman/libpod.
 
 ## Structure
 
-- **Basic Tests**
-  - [] output version number (Toolbox + Podman)
-  - [x] show help screen when no command is given
-  - [x] create the default container
-  - [x] create a container with a custom name
-  - [x] create a container from a custom image
-  - [x] list containers (no present)
-  - [x] list default container and default image
-  - [x] list containers (some present; different name patterns)
-  - [x] list images (no present)
-  - [x] list images (some present; different name patterns)
-  - [x] remove a specific container
-  - [x] try to remove nonexistent container
-  - [x] try to remove a running container
-  - [x] remove all containers
-  - [x] try to remove all containers (running)
-  - [x] remove a specific image
-  - [x] remove all images
-  - [x] run a command inside of an existing container
-
-- **Advanced Tests**
-  - [ ] create several containers with various configuration and then list them
-  - [ ] create several containers and hop between them (series of enter/exit)
-  - [ ] create a container, enter it, run a series of basic commands (id,
-        whoami, dnf, top, systemctl,..)
-  - [ ] enter a container and test basic set of networking tools (ping,
-        traceroute,..)
-
-The list of tests is stil rather basic. We **welcome** PRs with test
-suggestions or even their implementation.
+- **0xx (Info)**
+  - In this section are ran commands that are not dependent on the presence/number of containers/images (eg. getting version, help,..).
```

Comment:
Nightpick: would be nice to keep the documentation files limited to 80 characters per line.

When it comes to code, depending on the programming language, the ability to use more than 80 characters can be hugely convenient in practical terms. Human languages are usually easier.

Reply:
Tweaked.